### PR TITLE
AAP-1636: New AAP Hardening Guide

### PR DESCRIPTION
**Pull request by lmalivert**
Jira [AAP-1636](https://issues.redhat.com/browse/AAP-1636)

Created new guide - AAP Hardening Guide for RHEL

The assemblies that have been created represent each chapter of the [AAP Hardening Guide for RHEL](https://docs.google.com/document/d/1A6VLRNEzvKw4GWhcQpIMpSN9ylFpMGsGLeIUU-7p19I/edit#heading=h.3x04zlgkg9kv).

New directories:
- `/titles/aap-hardening` with symlink to `aap-common` `aap-hardening` `attritbutes` `image`
- `/assemblies/aap-hardening` with assembly files `assembly-aap-compliance` `assembly-aap-security-enabling` `assembly-hardening-aap` `assembly-intro-to-aap-hardening`
- `/modules/aap-hardening`

New content:
- `docinto.xml` `master.adoc`

### CODE REVIEW

File: downstream/titles/aap-hardening/master.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+include::attributes/attributes.adoc[]
+
+
+// Book Title
+= {PlatformName} Hardening Guide for RHEL
+
+This guide provides recommended practices for various processes needed to install, configure, and maintain {PlatformNameShort} on Red Hat Enterprise Linux in a secure manner.
+
+include::aap-on-azure/assembly-aap-compliance.adoc[leveloffset=+1]
```
Comment (ariordan-redhat) [inline downstream/titles/aap-hardening/master.adoc:15]:
```suggestion
include::aap-hardening/assembly-aap-compliance.adoc[leveloffset=+1]
```

**Review by ariordan-redhat — CHANGES_REQUESTED** [review]

**Comment by lmalivert** [comment]
Thanks for the review, @ariordan-redhat! I've committed the changes that you suggested. 
Please review and then squash and merge if it LGTY 👍🏾

File: downstream/titles/aap-hardening/docinfo.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<title>Red Hat Ansible Automation Platform Hardening Guide for RHEL</title>
+<productname>Red Hat Ansible Automation Platform</productname>
+<productnumber>2.4</productnumber>
+<subtitle>Install, configure, and maintain Ansible Automation Platform running on Red Hat Enterprise Linux in a secure manner.</subtitle>
+<abstract>
+    <para><emphasis role="strong">Providing Feedback:</emphasis></para>
+    <para> If you have a suggestion to improve this documentation, or find an error, please contact technical support at <link xlink:href="https://access.redhat.com"></link> to create an issue on the <emphasis role="strong">Ansible Automation Platform</emphasis> Jira project using the <emphasis role="strong">Docs</emphasis> component.</para>
```
Comment (ariordan-redhat) [inline downstream/titles/aap-hardening/docinfo.xml:7]: We're deleting this content from all abstracts in the docset

Comment (lmalivert) [reply downstream/titles/aap-hardening/docinfo.xml:7]: Sounds good, @ariordan-redhat 👍🏾 I'll note that down. Thanks!

**Pull request merged**